utils.py

```diff
@@ -2614,7 +2625,7 @@ def fetch_detox_info(self, talk=True, buffer_level=0.8, sites_space_override=Non
             #print site,self.queue.get(site,0)
             ## consider quota to be 80% of what's available
             queued_used = 0
-            available = int(float(quota)*buffer_level) - int(locked) - int(queued_used)
+            available = float(quota) - float(getGroupUsage(phedex_url,site))
```

Comment:
Did you check if the value are in the same unit (GB/TB)?

Reply:
dest_bytes is in bytes but quota is in TB. I will change both to the same unit.